src/lib/pbkdf/pbkdf.h

```diff
@@ -147,6 +161,20 @@ class BOTAN_DLL PBKDF
          }
    };
 
+/**
+* Password based key derivation function factory method
+* @param algo_spec the name of the desired PBKDF algorithm
+* @return pointer to newly allocated object of that type
+*/
+inline PBKDF* get_pbkdf(const std::string& algo_spec,
+                           const std::string& provider = "")
+   {
+   std::unique_ptr<PBKDF> p(PBKDF::create(algo_spec, provider));
+   if(p)
```

Comment:
Can we say that omitting braces is only allowed if the single statement is placed in the same line? [Style Conventions](https://github.com/randombit/botan/blob/master/doc/hacking.rst#style-conventions)